CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 67.0.0-SNAPSHOT - unreleased
 
+### 🎁 New Features
+
+* Enhanced `markdown` component to support a `components` prop to map standard html tags to
```

Comment:
Enhanced markdown component to support the underlying `components` prop from `react-markdown`.  Use this prop to customize markdown rendering.